docs/content/mobile/ios/cloud-deploy.md

```diff
@@ -13,6 +13,8 @@ Alternatively, if you want to create your own authentication server, follow [thi
 
 ## iOS Simulator{#ios-simulator}
 
+<%(Callout type="alert" message="The [**First Person Shooter (FPS) Starter Project**]({{urlRoot}}/projects/fps/overview) cannot currently run on the iOS Simulator. This is due to an incompatibility between the [Metal Graphics API](https://developer.apple.com/metal/) used by the project, and the Simulator.")%>
```

Comment:
We're trying to avoid using the callout boxes where possible because they're loud and obnoxious, like me. 
Suggest we change this to a note: 

```suggestion
**Note:** You cannot run the [First Person Shooter (FPS) Starter Project]({{urlRoot}}/projects/fps/overview) on the iOS Simulator. This is due to an incompatibility between the [Metal Graphics API](https://developer.apple.com/metal/) used by the project and the iOS simulator.
```

Is it worth adding this to the known issues page?

Reply:
yeah, would be good to put it there as well